.env

```diff
@@ -3,4 +3,4 @@ POSTGRES_PASS=zammad
 POSTGRES_USER=zammad
 RESTART=always
 # don't forget to add the minus before the version
-VERSION=-3.3.0-19
+VERSION=-3.4.0-1
```

Comment:
this version can't be predicted so we should wait for the first build and add the version here afterwards.
if somebody trys to update while build is still running he will get an "iamge not found error" too.

Reply:
👌 That means I should roll this back to `3.3.0-19` and wait for the output then?

Reply:
yes, remove changes from this file for now.